.github/workflows/ci-arm-build.yml

```diff
@@ -0,0 +1,67 @@
+name: CI ARM64 Build and Push
+
+on:
+  # push:
+  #   branches:
+  #     - "master"
+  #   tags-ignore:
+  #     - "*"
+  # pull_request:
+  #   branches-ignore:
+  #     - "*"
+  workflow_dispatch:
+
+jobs:
+  build-and-push-arm64:
+    if: github.event_name == 'workflow_dispatch'
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [ubuntu-22.04]
+        python: ["3.11"]
+    env:
+      # secrets can be set in settings/secrets on github
+      DOCKER_REGISTRY: ${{ secrets.DOCKER_REGISTRY }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: setup QEMU
```

Comment:
why qemu is needed? It is some sort of virtualizer? perhaps can add doc that in here within acomment?

Reply:
qemu is indeed a virtualizer. I guess github runenrs dont offer ARM so with this one can via hypervisor/KVM run arm stuff? just a guess

Reply:
ok technically not needed for the amd64 case. this was mostly for similarity with the other workflow. I will remove it from here.

Reply:
https://en.wikipedia.org/wiki/QEMU

Reply:
to get information, just ctrl+click on the action, in this case [https://github.com/docker/setup-qemu-action/tree/v3/](https://github.com/docker/setup-qemu-action/tree/v3/)